code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -68,6 +68,8 @@
 	if(..())
 		return 1
 
+	if(pulling && istype(pulling, /mob/living/carbon/human))
+		stop_pulling()

```

Comment:
https://github.com/ParadiseSS13/Paradise/pull/5801 fixes the Xeno organ runtime.